pegasus/sites.v3/hourofcode.com/i18n/en.yml

```diff
@@ -297,6 +298,8 @@
   module_label_educator_tool: "Educator tool"
   module_label_educator_course: "Educator course"
   module_label_student_tool: "Student tool"
+  module_label_student_product: "Product"
+  module_label_student_content_resources: "content & resources"
```

Comment:
Nit: can you update this to sentence case (`Content & resources`) so it's like the others?

Reply:
done